purchases/build.gradle.kts

```diff
@@ -149,6 +154,20 @@ fun obtainTestBuildType(): String =
         "debug"
     }
 
+// TO DO: Bring in Samsung SDK from somewhere else
+val samsungIapSdkPath = providers.provider {
+    providers.gradleProperty("samsungIapSdkPath").orNull
+        ?: providers.environmentVariable("SAMSUNG_IAP_SDK_PATH").orNull
+        ?: localProperties.getProperty("samsungIapSdkPath")
+}.map { path ->
+    val aar = file(path)
+    check(aar.exists()) {
```

Comment:
I guess this will fail to work if the aar is not set somehow... we shouldn't get this merged to main, but it's ok to keep it on the integration branch. Hopefully we can figure out a way to integrate it 🙏